Source/WebCore/platform/graphics/FontCascadeFonts.cpp

```diff
@@ -232,7 +233,7 @@ static bool shouldIgnoreRotation(UChar32 character)
     if (isInRange(character, 0x002E5, 0x002EB))
         return true;
 
```

Comment:
This file is cluttered with magic constants already, but we could start adding a comment explaining what is `0x018B0`.